Parsers/ASimFileEvent/ProductParsers/FileEventLinuxSysmonFileDeleted.yaml

```diff
@@ -0,0 +1,65 @@
+Parser:
+  Title: ASIM Sysmon/Linux File Deleted (Event Ids 23, 26) File Event Parser
+  Version: '0.1'
+  LastUpdated: July 27, 2021
+Product:
+  Name: Microsoft Linux Sysmon
+Normalization:
+  Schema: FileEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM File Schema
+  Link: https://aka.ms/AzSentinelFileEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AzSentinelNormalization
+Description: |
+  This is a Query Parser that is used to map Microsoft Linux Sysmon Event ids 23 and 26 (Event) to the Azure Sentinel Information Model FileEvent schema.
+ParserName: vimFileEventLinuxSysmonFileDeleted
+ParserQuery: |
+  Syslog
+    | where SyslogMessage has ('<Provider Name="Linux-Sysmon"') and SyslogMessage has_any('<EventID>23</EventID>','<EventID>26</EventID>')	
+    | parse SyslogMessage with  
+      '<Event><System><Provider Name="Linux-Sysmon" Guid="{ff032593-a8d3-4f13-b0d6-01fc615a0f97}"/><EventID>'msgEventId:string'</EventID>'
+      *
+      '<EventRecordID>'msgEventRecordID:string'</EventRecordID>'
+      *
+      '<Computer>'msgComputer:string'</Computer>'
+      '<Security UserId="'msgUserId:string'"/>'
+      *
+      '<Data Name="ProcessGuid">{'msgProcessGuid:string'}</Data>'
+      '<Data Name="ProcessId">'msgProcessId:string'</Data>'
+      '<Data Name="User">'msgUser:string'</Data>'
+      '<Data Name="Image">'msgImage:string'</Data>'
+      '<Data Name="TargetFilename">'msgTargetFilename:string'</Data>'
+      '<Data Name="Hashes">'msgHashes:string'</Data>' *	
+    | extend
+     EventCount=int(1)
+    , EventStartTime =TimeGenerated
+    , EventEndTime=TimeGenerated
+    , EventType = 'FileDeleted'
+    , EventResult ='Success' 
+    , EventProduct='Sysmon For Linux'
+    , EventProductVersion='v13.22' 
+    , EventVendor ='Microsoft'
+    , EventSchemaVersion ='0.1.0'
+    , DvcOs = 'Linux'
+    , ActorUserIdType='UID' /// 
+    , TargetFilePathType='Unix'
+    , ActorUsernameType='Simple'
+  | project-rename
+    EventMessage=SyslogMessage
+    , DvcHostname=Computer
+    , EventOriginalUid=msgEventRecordID
+    , EventOriginalType =msgEventId 
+    , ActorUserId=msgUserId
```

Comment:
While for delete there is a username, User ID is still not relevant and should be removed.